caas/broker.go

```diff
@@ -156,13 +156,13 @@ type Broker interface {
 	// ProviderRegistry is an interface for obtaining storage providers.
 	storage.ProviderRegistry
 
-	// ConstraintsChecker provides a means to check that constraints are valid.
```

Comment:
Why is this one deleted? It's needed when we deploy k8s charms

Reply:
coz `environs.BootstrapEnviron` has it already